.github/workflows/lock.yml

```diff
@@ -1,4 +1,4 @@
-name: 'Lock Threads'
+name: Lock Threads
```

Comment:
Could you append "Cron: " to this? The filename should probably also be renamed too.

Reply:
Maybe will rename to `cron-lock`

Reply:
file renamed, As for names itself, we can just remove `name` and github will use the filename instead, will try that.

Reply:
ah no, it will use ugly names, will set `name` to filename

<img width="629" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/3cb8f46f-879c-4f8b-9f7d-c3784384697f">

Reply:
Ok, looking good now

<img width="608" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/170517d9-fe26-4f10-9a2c-85daf2a5861e">